index.html

```diff
@@ -0,0 +1,857 @@
+                ><i class="fas fa-exclamation-triangle"></i>
+                <span class="rick">Database +3</span></a
+              >
+              <a href=""
+                ><i class="fas fa-briefcase"></i>
+                <span class="rick">BBC Ventures Capital +4</span></a
+              >
+            </div>
+            <ul class="clinchy-ul">
+              <li>
+                <a href="#" class="rick"
+                  >Canadian data start-up Cinchy lands $10m Series A on FinTech
+                  Futures</a
+                >
+              </li>
+            </ul>
+          </div>
+          <!-- +++++++++++++++++++++++++++++Cinchy3+++++++++++++++++++++++++++++++++ -->
+          <div class="cinchy">
+            <a href="#" class="cinchy-anchor">
+              <img src="images/l-5.jpg" class="cinchy-img" alt="cinchy img" />
+              <h2>OnConinvent As</h2>
+              <div class="series-div">
+                <h4>$10M</h4>
+                <h5>series a</h5>
+              </div>
+            </a>
+            <div class="below-cinchy">
+              <a href="#"
+                ><i class="fas fa-exclamation-triangle"></i>
+                <span class="rick">Database +3</span></a
+              >
+              <a href=""
+                ><i class="fas fa-briefcase"></i>
+                <span class="rick">BBC Ventures Capital +4</span></a
+              >
+            </div>
+            <ul class="clinchy-ul">
+              <li>
+                <a href="#" class="rick"
+                  >Canadian data start-up Cinchy lands $10m Series A on FinTech
+                  Futures</a
+                >
+              </li>
+            </ul>
+          </div>
+          <!-- ++++++++++++++++++++++++++++++Cinchy4_++++++++++++++++++++++++++++++++ -->
+          <div class="cinchy cinchy4">
+            <a href="#" class="cinchy-anchor">
+              <img src="images/l-5.jpg" class="cinchy-img" alt="cinchy img" />
+              <h2>Aircall</h2>
+              <div class="series-div">
+                <h4>$10M</h4>
+                <h5>series a</h5>
+              </div>
+            </a>
+            <div class="below-cinchy">
+              <a href="#"
+                ><i class="fas fa-exclamation-triangle"></i>
+                <span class="rick">Database +3</span></a
+              >
+              <a href=""
+                ><i class="fas fa-briefcase"></i>
+                <span class="rick">BBC Ventures Capital +4</span></a
+              >
+            </div>
+            <ul class="clinchy-ul">
+              <li>
+                <a href="#" class="rick"
+                  >Canadian data start-up Cinchy lands $10m Series A on FinTech
+                  Futures</a
+                >
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        APP SECTION
```

Comment:
- [ ] The text in the images are not in the correct position for medium and small screens. Kindly fix that.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83334131-09a0fd00-a26a-11ea-8cbf-bdb92b642df6.png) | ![image](https://user-images.githubusercontent.com/57414789/83334119-f726c380-a269-11ea-9aee-52abd9607f21.png)